include/config_defaults.h

```diff
@@ -236,6 +236,10 @@
 #define SERVICE_BLE_COMPANION 0
 #endif
 
+#ifndef SERVICE_NOTIFIER
+#define SERVICE_NOTIFIER 1
+#endif
```

Comment:
You are adding a feature to the OS? Please prefix it with `OSW_` accordingly (I know we are not consistent with the old features, but we don't want to break compatibility either).